src/net.h

```diff
@@ -1167,6 +1167,13 @@ class CConnman
     friend struct ConnmanTestMsg;
 };
 
+/**
+ * Get a unique identifier for the network (e.g. Tor, IPv4/6) of a connection.
```

Comment:
nit: From my reading of the first line of the comment, it looks like this is just a network type identifier (without the local socket). e.g., tor=0, ipv4=1, etc.